src/core/FinderFacade.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\Tombstone\Core;
+
+use Symfony\Component\Finder\Finder;
+
+final class FinderFacade
+{
+    private $items = [];
+
+    private $excludes = [];
+
+    private $names = [];
+
+    private $notNames = [];
+
+    public function __construct(array $items = [], array $excludes = [], array $names = [], array $notNames = [])
+    {
+        $this->items = $items;
+        $this->excludes = $excludes;
+        $this->names = $names;
+        $this->notNames = $notNames;
+    }
+
+    /**
+     * @return string[]
+     */
+    public function findFiles(): array
```

Comment:
Would you please add test cases for this method? I'd be fine taking over the test cases from the original `sebastianbergmann/finder-facade` library.